server/src/main/java/org/opensearch/throttling/admissioncontrol/utils/AdmissionControlActionsMap.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.throttling.admissioncontrol.utils;
+
+import org.opensearch.throttling.admissioncontrol.enums.TransportActionType;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Map to store the actions based on the type of the transport operation
+ */
+public class AdmissionControlActionsMap {
+    public static Map<String, List<String>> transportActionsMap;
+
+    // Adding below as actions will add/update more actions based on discussions in next PR's.
+    static {
+        transportActionsMap = new HashMap<>();
+        transportActionsMap.put(TransportActionType.INDEXING_PRIMARY.getType(), List.of("indices:data/write/bulk[s][p]"));
+        transportActionsMap.put(TransportActionType.INDEXING_REPLICA.getType(), List.of("indices:data/write/bulk[s][r]"));
+        transportActionsMap.put(
+            TransportActionType.SEARCH_QUERY.getType(),
+            Arrays.asList(
+                "indices:data/read/search[phase/query]",
+                "indices:data/read/search[phase/query/id]",
+                "indices:data/read/search[phase/dfs]"
+            )
+        );
+        transportActionsMap.put(TransportActionType.SEARCH_FETCH.getType(), List.of("indices:data/read/search[phase/fetch/id]"));
+    }
```

Comment:
why do we need to hardcode string?

Reply:
Updated String values with actual action variables